src/stores/accountDetails.js

```diff
@@ -1,6 +1,6 @@
 import { defineStore, storeToRefs } from 'pinia'
-import axios from 'axios'
 import { useRuntimeConfig } from 'nuxt/app'
+import useAxios from '@/composables/useAxios'
```

Comment:
Would it be somehow possible to destruct the variable from it, so we can save one line per store?

Reply:
I see now, thanks for explanation